cmd/pinniped-proxy/build.rs

```diff
@@ -0,0 +1,9 @@
+use std::env;
+
+fn main() {
+    // If the binary is built with the ENV var "PINNIPED_PROXY_VERSION",
+    // the value will be available at buildime. Otherwise, it becomes "devel"
+    // We use this ENV var to display a custom version when passing the "-- version" flag
+    let foo = env::var("PINNIPED_PROXY_VERSION").unwrap_or("devel".to_string());
```

Comment:
s/foo/version/ ?

And nice, hadn't read about `bulid.rs` - interesting interface to have for the build environment.

Reply:
Yep, right.

It is really interesting (https://doc.rust-lang.org/cargo/reference/build-scripts.html), it seems to be very promising. I still have to take a closer look a the docs, just to see if there is something else we can leverage from.